tests/console/journalctl.pm

```diff
@@ -205,6 +205,12 @@ sub run {
     assert_script_run 'journalctl --flush';
     verify_journal();
     # Write second series of messages with different log priority
+
+    if (is_sle('=15') && is_s390x) {
+        zypper_call 'in haveged';
+        systemctl 'start haveged';
```

Comment:
i think it would help a comment here of why you install haveged. Also i wonder if you should check the `/proc/sys/kernel/random/entropy_avail` and act based on the value there, rather than make it specific. It is up to you either way.

Reply:
I was not aware of `/proc/sys/kernel/random/entropy_avail`, thanks! Frankly, it is such an isolated problem, that I am not sure whether it is worth to overcomplicate it. Mainly, I have no idea what should be the threshold that would guarantee that the key generates within default openqa timeout.

Reply:
as i said, just an idea for thought. as you said lets not overcomplicate it too